client/src/views/TableDetail.vue

```diff
@@ -52,7 +61,7 @@ export default {
   
       let table = response.data.data.tables[0];
       this.rowKeys = table.rows.rows.map(r=> r.columns.filter(c=> c.key != "_rev"))
-     
+      this.headers = this.rowKeys[0].map(k=> k.key.startsWith("_") ? k.key.split("_")[1] : k.key);
```

Comment:
That `k.key.split("_")[1]` looks dangerous to me--it won't work properly if a key name has an underscore somewhere in the middle.

If the intent is just to strip off a leading underscore, then I think `k.key.slice(1)` will work better.